### PR TITLE
Calendar UI #44

### LINKED ISSUE #44: Calendar UI
Improving Calendar UI

### PR DESCRIPTION
**Pull request by joegery**
closes #44

### CODE REVIEW

**Comment by joegery** [comment]
Waiting until #24 is merged to implement times

**Comment by joegery** [comment]
- Added Weekly Calendar View
- Improved Calendar UI
- Added functionality to view tasks on calendar day after clicking on the day
- Added a highlight on days with multiple tasks to show how many tasks are on that day

**Review by StephenRhoades — APPROVED** [review]

**Review by Dbschmidle — APPROVED** [review]
Nice 👍

**Pull request merged**
